### ISSUE TITLE
This should be NPM installable and not a Django App

### ISSUE THREAD

**Issue by glassresistor** [PaulUithol/backbone-tastypie#32]
Since there is no python besides setup.py and **init**.py why isn't this just an adaptor for backbone to tastypie rather than the other way around?

**Comment by nick-allen** [comment]
Agreed, this has almost no reason to be a Django app accept that it happens to be a connector for django-tastypie.

**Comment by glassresistor** [comment]
wonder if its worth maybe building an npm adapter for static files?

On Wed, Aug 7, 2013 at 12:29 PM, nick-allen notifications@github.comwrote:

> Agreed, this has almost no reason to be a Django app accept that it
> happens to be a connector for django-tastypie.
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/PaulUithol/backbone-tastypie/issues/32#issuecomment-22277211
> .

## 

Mikela

**Comment by PaulUithol** [comment]
There's a `package.json` in there now, along with a `bower.json`. That makes it NPM installable I think? And it still functions as a django app as well.

**Comment by c089** [comment]
It's installable in a way, but it would be even better if you could publish the released version to the npm registry, so users can install with the package name and version number instead of pointing to the repository.

**Comment by Robinfr** [comment]
It's not NPM installable since it was not released on NPM https://www.npmjs.com/search?q=backbone-tastypie

**Comment by tutuca** [comment]
Just hitting the need to have this on NPM to update my project that relies on this package.
